src/Zynga/Framework/IO/Web/V1/Manager.hh

```diff
@@ -50,4 +53,25 @@ class Manager {
       throw new UnexpectedHttpCodeException((string)$returnCode);
     }
   }
+  
+  public static function makeCurlRequest(
+    UrlBox $uploadUrl,
+    Map<string, mixed> $postParams): array<string, mixed> {
+      if(self::$useMockCurl == false) {
+        $session = curl_init();
```

Comment:
You need to check if this returns 'false'

http://php.net/manual/en/function.curl-init.php